dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/suite/FATSuite.java

```diff
@@ -80,7 +80,7 @@ public static RepeatTests telemetry10and11Repeats(String serverName) {
     }
 
     public static RepeatTests telemetry20Repeats(String serverName) {
-        return TelemetryActions.repeat(serverName, TelemetryActions.MP14_MPTEL20, TelemetryActions.MP41_MPTEL20, TelemetryActions.MP50_MPTEL20, TelemetryActions.MP60_MPTEL20, TelemetryActions.MP61_MPTEL20);
+        return TelemetryActions.repeat(serverName, TelemetryActions.MP14_MPTEL20, TelemetryActions.MP41_MPTEL20, TelemetryActions.MP50_MPTEL20, MicroProfileActions.MP70_EE10, MicroProfileActions.MP70_EE11);
```

Comment:
MP Tel 2.0 should run MP70_EE10 in LITE mode ... i.e. that one should be listed first. However, we can fix that in a follow up PR.